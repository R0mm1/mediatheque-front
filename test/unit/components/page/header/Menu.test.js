import {mount} from '@vue/test-utils'
import Menu from "@/components/pages/header/Menu.vue"

describe('Menu', ()=>{
    const wrapper = mount(Menu);

    it('renders correctly', ()=>{
        expect(wrapper.element).toMatchSnapshot();
    });
});
