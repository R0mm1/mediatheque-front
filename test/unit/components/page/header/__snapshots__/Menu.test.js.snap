// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<ul
  id="mainMenu"
>
  <li>
    
        Livres
        
    <ul
      class="subMenu l1Menu"
    >
      <li>
        <a
          href="/book"
        >
          Par livre
        </a>
      </li>
      <li>
        <a
          href="/author"
        >
          Par auteur
        </a>
      </li>
    </ul>
  </li>
</ul>
`;
